src/main/java/app/gpx_animator/core/Help.java

```diff
@@ -94,8 +95,9 @@ public static void printHelp(final OptionHelpWriter w) {
         w.writeOptionHelp(Option.VIEWPORT_HEIGHT, "viewport-height", false, cfg.getViewportHeight()); //NON-NLS
         w.writeOptionHelp(Option.VIEWPORT_INERTIA, "viewport-inertia", false, cfg.getViewportInertia()); //NON-NLS
         w.writeOptionHelp(Option.WAYPOINT_SIZE, "size", false, cfg.getWaypointSize()); //NON-NLS
-        w.writeOptionHelp(Option.WIDTH, "width", false, "(800)"); // TODO cfg.getWidth() NON-NLS
+        w.writeOptionHelp(Option.WIDTH, "width", false, cfg.getWidth()); //NON-NLS
```

Comment:
If I remember correctly, there was a problem using `getWidth()` based on if the width was set by the user or calculated programmatically. We should test that very well.